Automattic/woocommerce-payments
# fix: size of payment methods in modal

Fixes #

#### Changes proposed in this Pull Request








Current:
![Screen Shot 2021-07-23 at 5 21 10 PM](https://user-images.githubusercontent.com/273592/126846854-b460c1fe-0757-4bbe-8bb9-1f411180a1f3.png)

Designs:
![Screen Shot 2021-07-23 at 5 20 18 PM](https://user-images.githubusercontent.com/273592/126846791-e897bb70-5ed9-41e0-a69f-db0a0bc33e5d.png)

After:
![Screen Shot 2021-07-23 at 5 20 43 PM](https://user-images.githubusercontent.com/273592/126846826-29201c4f-424b-4120-963f-41f29a4c5ee9.png)

Icon border can be added by https://github.com/Automattic/woocommerce-payments/issues/2544

#### Testing instructions





- http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Enable UPE
- Add some payment methods
- Disable UPE via the hamburger menu
- Modal should be updated

-------------------
